src/poly.rs

```diff
@@ -200,3 +200,32 @@ impl<'a, F: Field, B: Basis> Mul<F> for Polynomial<F, B> {
         self
     }
 }
+
+/// Describes a relative location in the evaluation domain; applying a rotation
```

Comment:
Now that we're using this in two different ways, a more generic description might be:

```
/// Describes the relative rotation of a vector. Negative numbers represent
/// reverse (leftmost) rotations and positive numbers represent forward (rightmost)
/// rotations. Zero represents no rotation.
```